homeassistant/components/recorder/__init__.py

```diff
@@ -138,13 +142,23 @@ def run_information(hass, point_in_time: Optional[datetime] = None):
     if not db_url:
         db_url = DEFAULT_URL.format(hass_config_path=hass.config.path(DEFAULT_DB_FILE))
 
+    db_max_retries = conf.get(CONF_DB_MAX_RETRIES, None)
+    if not db_max_retries:
+        db_max_retries = DEFAULT_DB_MAX_RETRIES
+
+    db_retry_wait = conf.get(CONF_DB_RETRY_WAIT, None)
+    if not db_retry_wait:
+        db_retry_wait = DEFAULT_DB_RETRY_WAIT
```

Comment:
Remove the if statements because they are not needed if a default value is set.

Reply:
Implemented the suggested change in 8bac36bdd014bd9f2f5299aae40019bf23ea6ad7